UIResText.cs

```diff
@@ -23,7 +24,7 @@ public UIResText(string caption, Point position, float scale, Color color)
         }
 
         public UIResText(string caption, Point position, float scale, Color color, Font font, Alignment justify)
-            : base(caption, position, scale, color, font, false)
+            : base(caption, position, scale, color, font, 0)
```

Comment:
I don't know how the last variable changed, it would not accept a boolean with resharper so I just used 0.